packages/apps/frontera/src/routes/src/components/CommandMenu/commands/shared/DeleteConfirmationModal.tsx

```diff
@@ -29,12 +29,24 @@ export const DeleteConfirmationModal = observer(() => {
 
   const handleClose = () => {
     store.ui.commandMenu.toggle('DeleteConfirmationModal');
+    store.ui.commandMenu.clearCallback();
   };
 
+  const oppoortunityOfOrgSelected = store.opportunities
```

Comment:
can this be moved to one of the .with conditions? it's domain specific

Reply:
i need that for organization so would you think it will make more sens to be in with('Organizations'...) and more than this is just one object :D i don't see any concern regarding this constant